build_deploy.sh

```diff
@@ -2,6 +2,20 @@
 
 set -exv
 
+BACKWARDS_COMPATIBILITY="${BACKWARDS_COMPATIBILITY:-enabled}"
+LOCAL_BUILD="${LOCAL_BUILD:-false}"
+IMAGE_REGISTRY="quay.io"
+IMAGE_REPOSITORY="cloudservices/compliance-backend"
```

Comment:
Maybe a suggestion, if thes can be also set by an env var so we can push to other repos.

Reply:
Makes sense, I'll add it, as well as for the `IMAGE_REGISTRY`